scripts/prReleaseNotesCommon.js

```diff
@@ -99,12 +100,14 @@ function getPRsByType(PRs) {
       fixes.push(pr);
     } else if (pr.branch.startsWith('infra/')) {
       infra.push(pr);
+    } else if (pr.branch.startsWith('assets/') || pr.branch.startsWith('Assets/')) {
```

Comment:
instead of checking both `assets` and `Assets` you can use toLowerCase on the branch name and check only against `assets`